### DISCUSSION TITLE
Best LORAs for hands and legs?

### DISCUSSION THREAD

**Discussion by snoweeethecat** [lllyasviel/Fooocus#1824] [General]
Many times, the images generated have distorted hands, legs, fingers and toes.
I have already added all the necessary negative prompts, but deformities continue to exist.

Are there any recommendations for LORAs that can improve the hands and legs?
Thank you.

**Comment by thiner** [comment]
Maybe you can try Inpaint.
![image](https://github.com/lllyasviel/Fooocus/assets/1897227/832cc487-ed14-4697-b70d-8dc332e5b821)

**Comment by snoweeethecat** [comment]
> Maybe you can try Inpaint. 


What about this LORA shown in below link? 

What is the top recommended hands and legs LORA at CivitAI and/or Huggingface?

[https://civitai.com/models/121253/ai-infinity-realistic-better-hands](url)

**Reply by thiner** [reply]
you can put lora file into `models/loras` folder, and then select the lora in `Model` tab.
